### DISCUSSION TITLE
"Become a sponsor to `@user`" shows up another user's profile picture

### DISCUSSION THREAD

**Discussion by Malix-Labs** [community/community#75868] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

"Become a sponsor to `@user`" shows up another user's profile picture

## Exemple

[Mine](https://github.com/sponsors/Malix-off) shows up @alix's profile picture

**Comment by anjuan** [comment]
Hi @Malix-off, I'm not seeing the profile picture of the person you mentioned. Were you able to resolve this?

**Reply by Malix-Labs** [reply]
> Hi @Malix-off, I'm not seeing the profile picture of the person you mentioned. Were you able to resolve this?

Hi @anjuan, 
Nope, it has always been like that, in multiple browser
